pkg/agent/config/node_config.go

```diff
@@ -117,6 +117,10 @@ type AdapterNetConfig struct {
 }
 
 type WireGuardConfig struct {
+	// Enable shows that WireGuard is enabled or not.
+	Enable bool
+	// Initialized shows the WireGuard interface is Initialized or not.
```

Comment:
I feel we should not add "Enable" and "Initialized" to this struct, as they are just used internally in `mc_route_controller`(?). Probably they should just be members of `MCRouteController`.